client/src/common/utils/helpers.js

```diff
@@ -20,7 +20,14 @@ export const makeAbortablePromise = promise => {
 
 export const dateFromString = string => new Date(string).toLocaleString();
 
-export const validatePassword = value => value.match(/^[^\s]{4,32}$/);
+export const validatePassword = value =>
```

Comment:
Please just check the length of the trimmed input:
```javascript
const trimmed = value.trim();
return trimmed.length >= MIN && trimmed.length <= MAX;
```

Reply:
We don't want the user to put spaces in their password.